perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -173,27 +173,26 @@
     <arg name="input/object0" value="$(var merger/input/objects)"/>
     <arg name="input/object1" value="clustering/camera_lidar_fusion/short_range_objects"/>
     <arg name="output/object" value="camera_lidar_fusion/objects"/>
-    <arg name="node_name" value="object_association_merger_camera_lidar_fusion" />
+    <arg name="node_name" value="object_association_merger_camera_lidar_fusion"/>
   </include>
   <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
     <arg name="input/object0" value="detection_by_tracker/objects"/>
     <arg name="input/object1" value="camera_lidar_fusion/objects"/>
     <arg name="output/object" value="temporary_merged_objects"/>
-    <arg name="node_name" value="object_association_merger_detection_by_tracker" />
+    <arg name="node_name" value="object_association_merger_detection_by_tracker"/>
   </include>
   <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
     <arg name="input/object0" value="temporary_merged_objects"/>
     <arg name="input/object1" value="clustering/objects"/>
     <arg name="output/object" value="objects_before_filter"/>
-    <arg name="node_name" value="object_association_merger_unknown_clustering" />
+    <arg name="node_name" value="object_association_merger_unknown_clustering"/>
   </include>
 
   <!-- Filter -->
   <group>
     <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
       <arg name="input/object" value="objects_before_filter"/>
       <arg name="output/object" value="$(var output/objects)"/>
-      <arg name="filtering_range_param" value="$(find-pkg-share perception_launch)/config/object_recognition/detection/object_lanelet_filter.param.yaml" />
```

Comment:
same as https://github.com/tier4/autoware_launch/pull/408/files#r918585971

Reply:
This line should remain. I will fix tier4_perception_launch.

Reply:
changed at [ada8e61](https://github.com/tier4/autoware_launch/pull/408/commits/ada8e61706487ee44d9aec16aff9c409a27c4e46)